jenkins/master/configuration/init.groovy.d/ods-mro-jenkins-shared-library.groovy

```diff
@@ -0,0 +1,63 @@
+#!groovy
+
+// imports
+import hudson.scm.SCM
+import jenkins.model.Jenkins
+import jenkins.plugins.git.GitSCMSource
+import org.jenkinsci.plugins.workflow.libs.*
+import org.jenkinsci.plugins.workflow.libs.LibraryConfiguration
+import org.jenkinsci.plugins.workflow.libs.SCMSourceRetriever
+
+def namespace = "cat /var/run/secrets/kubernetes.io/serviceaccount/namespace".execute().text.trim()
+println "INFO: Jenkins adding mro shared lib: ${namespace}"
+
+def buildSharedLibName = "ods-jenkins-shared-library"
+
+def env = System.getenv()
+def buildSharedLibraryRepository = env['SHARED_LIBRARY_REPOSITORY']
+
+println "INFO: Jenkins adding mro shared lib - build lib path ${buildSharedLibraryRepository}"
+
+def mroLibRepoPath = buildSharedLibraryRepository.replace(buildSharedLibName, "ods-mro-jenkins-shared-library")
+
+println "INFO: Jenkins adding mro shared lib path: ${mroLibRepoPath}"
+
+def credentialsId = namespace + "-cd-user-with-password"
+// parameters
+def globalLibrariesParameters = [
+  branch:               "master",
```

Comment:
Should this be `master` or `production`?

Reply:
@michaelsauter  - catch! and one can override it later in the real usage ... but following the design of ODS I point it to production :)